python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -268,7 +268,7 @@ def run_optimize_model(repo_id,
         model = optimize_model(model)
         tokenizer = LlamaTokenizer.from_pretrained(model_path, trust_remote_code=True)
     else:
-        model = AutoModelForCausalLM.from_pretrained(model_path, torch_dtype='auto', low_cpu_mem_usage=True)
+        model = AutoModelForCausalLM.from_pretrained(model_path, torch_dtype='auto', low_cpu_mem_usage=True, trust_remote_code=True,use_cache=True) #XXX
         model = optimize_model(model)
         tokenizer = AutoTokenizer.from_pretrained(model_path)
     end = time.perf_counter()
```

Comment:
Let's raise another PR for all-in-one benchmark fixes. Do not make one PR too big/complex

Reply:
Have updated , and this PR now is only for adding example for Qwen-VL-Chat.